docs/Gestor_Dependencias.md

```diff
@@ -1,10 +1,28 @@
 # Elección del gestor de dependencias.
 
+
+
 ## Para qué se necesita un gestor de dependencias.
 
 Como bien se menciona [aquí](https://ibm.github.io/data-science-best-practices/dependency_management.html)
 > Dependency management is like your city’s sewage system. When it’s working well, it’s easy to forget that it even exists. The only time you’ll remember it is when you experience the agony induced by its failure.
 
+Teniendo en cuenta el [standard de la estructura de los proyectos en Python](https://docs.python-guide.org/writing/structure/) y [este ejemplo de estructura de proyectos](https://packaging.python.org/en/latest/tutorials/packaging-projects/).
+
+La estructura que buscamos tener sería:
+
+- proyecto
+    - LICENSE
+    - pyproject.toml
+    - README.md
+    - src/
```

Comment:
Esto es lo que recomienda el segundo; el primero recomienda otra cosa. Y si usas poetry y lo usas para generar la estructura del proyecto, [te creará otra cosa diferente](https://mathspp.com/blog/how-to-create-a-python-package-in-2022)